### DISCUSSION TITLE
How to install a nightly build?

### DISCUSSION THREAD

**Discussion by lz1998** [coder/code-server#4142] [Q&A] [answered]
I can't use git graph in `3.11.0` and `3.11.1`, because css is incorrect.

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
As far as I know, there are no nightly builds. You will have to build the entire code from the source to get the latest commit up and running.

**Reply by jsjoeio** [reply]
Yes, unfortunately, this is correct. This issue is fixed [here](https://github.com/cdr/code-server/pull/4131) and will be out in `3.12.0`. 

You could download the `release-packages` [here](https://github.com/cdr/code-server/runs/3562652382) and run code-server from there but that might be more complicated and more trouble than it's worth

![image](https://user-images.githubusercontent.com/3806031/133126925-830bb763-546b-4c7d-a1e1-813ac6c9d67d.png)
